components/eamxx/scripts/machines_specs.py

```diff
@@ -202,9 +202,12 @@ class Compy(Machine):
     @classmethod
     def setup(cls):
         super().setup_base("compy")
-
-        cls.env_setup = ["module purge", "module load cmake/3.19.6 gcc/8.1.0  mvapich2/2.3.1 python/3.7.3"]
-        cls.batch = "srun --time 02:00:00 --nodes=1 -p short --exclusive --account e3sm"
+        compiler = "intel"
```

Comment:
There is no `compiler` entry in the base class. Of course, you can add one, if you think it helps clarify things, but be aware that this is not used anywhere.